### PR TITLE
Bug/statistics labels

### PR DESCRIPTION
**Pull request by Brennan1994**

### CODE REVIEW

**Comment by Brennan1994** [comment]
This refactor moves us from using the Statistics Enum to a string based statistics label in both DSS and SQLite

**Review by HenryGeorgist — APPROVED** [review]
looks good to me.

File: FIRO_TSEnsembles/src/main/java/hec/SqliteDatabase.java [hunk 1]
```diff
@@ -676,12 +676,12 @@ private MetricCollection getMetricCollection(ResultSet rs) throws SQLException {
         byte[] byte_value_array = rs.getBytes("byte_value_array");
 
         float[][] values = TableCompression.UnPack(byte_value_array, member_count, member_length, compression);
-        String[] params = metricstats.split(",");
-        Statistics[] stats = new Statistics[params.length];
-        for (int i = 0; i < params.length; i++){
-            stats[i] = Statistics.valueOf(params[i]);
-        }
-        return new MetricCollection(issue_date, start_date, stats, values); //Duration.ofSeconds(interval_seconds),units);
+//        String[] params = metricstats.split(",");
```
Comment (ktarbet) [inline FIRO_TSEnsembles/src/main/java/hec/SqliteDatabase.java:679]: Would be nice if the commented lines are removed.

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MaxAccumDuration.java [hunk 2]
```diff
@@ -66,4 +66,9 @@ public void configure(Configuration c) {
     public Statistics[] Statistics() {
         return new Statistics[]{Statistics.MAXACCUMDURATION};//but what duration?
     }
+
+    @Override
+    public String StatisticsLabel() {
```
Comment (ktarbet) [inline FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MaxAccumDuration.java:71]: Nice Name!

File: FIRO_TSEnsembles/src/main/java/hec/metrics/MetricCollection.java [hunk 3]
```diff
@@ -12,40 +12,37 @@ public class MetricCollection {
...
+        this(new MetricsConfiguration(issueDate,startDate), statisticsLabel, values);
     }
 
-    public Statistics[] getMetricStatistics() {
-        return metric_statistics;
+    public  String getMetricStatistics() {
+        return metric_statisticsLabel;
     }
 
+    //TODO: fix this to be specific to string label. Contains will catch on first instance every time if multiple.
```
Comment (ktarbet) [inline FIRO_TSEnsembles/src/main/java/hec/metrics/MetricCollection.java:32]: I don't see any usages of this method.  Perhaps used by a plugin?

File: ensemble-dss/src/main/java/hec/dss/ensemble/DssDatabase.java [hunk 4]
```diff
@@ -35,22 +35,32 @@
  */
 public class DssDatabase implements EnsembleDatabase,MetricDatabase {
     private String dssFileName;
-    Catalog catalog;
+    private Catalog catalog;
     static DateTimeFormatter dssDateFormat = DateTimeFormatter.ofPattern("ddMMMyyyy HHmm");
```
Comment (ktarbet) [inline ensemble-dss/src/main/java/hec/dss/ensemble/DssDatabase.java:39]: Can the three DateTimeFormatters here be make private?

File: ensemble-dss/src/main/java/hec/dss/ensemble/MetricPathTools.java [hunk 5]
```diff
@@ -40,6 +41,14 @@ static Statistics getMetricStatFromPath(String toString) {
         return null;
     }
 
+    static String getMetricStatLabelFromPath(DSSPathname path) {
```
Comment (ktarbet) [inline ensemble-dss/src/main/java/hec/dss/ensemble/MetricPathTools.java:44]: Nice to have a method for this!

**Review by ktarbet — APPROVED** [review]
Approved - I added with some ideas to consider .

File: FIRO_TSEnsembles/src/main/java/hec/metrics/MetricCollection.java (hunk 3 above)
Comment (Brennan1994) [reply FIRO_TSEnsembles/src/main/java/hec/metrics/MetricCollection.java:32]: I don't really understand it's use case. @HenryGeorgist is there a reason to keep this method around?

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MaxAccumDuration.java (hunk 2 above)
Comment (Brennan1994) [reply FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MaxAccumDuration.java:71]: Thanks!

File: FIRO_TSEnsembles/src/main/java/hec/SqliteDatabase.java (hunk 1 above)
Comment (Brennan1994) [reply FIRO_TSEnsembles/src/main/java/hec/SqliteDatabase.java:679]: done

File: ensemble-dss/src/main/java/hec/dss/ensemble/DssDatabase.java (hunk 4 above)
Comment (Brennan1994) [reply ensemble-dss/src/main/java/hec/dss/ensemble/DssDatabase.java]: yep!

**Pull request merged**